### ISSUE TITLE
API service

### ISSUE THREAD

**Issue by viktor-yakubiv** [voveus/weather-app#1]
To develop our application we need an API. 

You should choose one and develop a service, for example `services/api.js`. It may be a class, singleton object or just list of functions.

Service responsibilities:

- get coordinates by query
- get forecast by coordinates for today/tomorrow/next week

You may suggest you own API methods, it does not matter. However, reply to this issue which methods you decided to expose from API to prevent blocking of development other tasks.

**Comment by LiliVia** [comment]
I try to do this task.

**Comment by viktor-yakubiv** [comment]
Please, describe the interface of your class. Which methods it exposed. It helps us to prevent blocking of other tasks.

**Comment by viktor-yakubiv** [comment]
We need to pass city or location, not only date.

**Comment by LiliVia** [comment]
```
getForecast(city = currentLocation, start = new Date, end)
getDailyForecast(city = currentLocation, date = new Date)
getWeeklyForecast(city = currentLocation, start = new Date)
```

**Comment by Boyko-Anna** [comment]
Why do we have to pass date as a parameter to these methods? I thought the only thing that changes in our requests are the city and the Fahrenheit/Celsius units... Do I misunderstand something?

**Comment by Boyko-Anna** [comment]
Or even to pass only the city with Celsius units as default and then calculate Fahrenheit without making additional request...

**Comment by Boyko-Anna** [comment]
@viktor-yakubiv And one more question... In this issue we have: "get forecast by coordinates", but in Openweathermap we find: "We recommend to call API by city ID"... Is there any significant difference between these two? 
And sorry for so many questions...

**Comment by viktor-yakubiv** [comment]
I do not know API well. Feel free to choose any methods signature you want. Anyway, we need to find city ID somehow.

Yes, calculation of temperature is better than one more API call.

**Comment by Boyko-Anna** [comment]
Ok. Thanks) They offer json file with cities IDs to download.

**Comment by LiliVia** [comment]
In this case we can use only one request by city name or id to get 5 days forecast if we use openweather API or 16 days forecast wich free on weatherbit.io. 
And then calculate all needed info.

**Comment by viktor-yakubiv** [comment]
Resolved by #6
